SixTrack/sixtrack.s

```diff
@@ -3890,50 +3892,55 @@
           pi=4d0*atan(1d0)
 +ei
         crabfreq=ek(ix)*c1e3
-
+        crabamp=ed(ix)
+                                 
         do j=1,napx ! loop over particles
-         crabamp=(ed(ix)/ejfv(j))*c1e3                                   !hr03
 
+         kcrab=(((sigmv(j)/(clight*(e0f/e0)))*crabfreq)*2d0)*pi         &!hr03
+     & +  crabph(ix)                                                     !hr03 
 +if .not.tilt
```

Comment:
Just nuke the +if tilt / +if .not.tilt already OR implement the tilt; the as long as the code is actually the same, having the `+if` is just misleading...

Reply:
implementing tilt requires also a review on the madx side, unfortunately. I would implement the tilt when we have a generic rf-multipole and review madx conversion at the same time. One can remove the "if tilt" at the sixtrack V merging stage.

Reply:
I cannot comment the code below for some reason.  I am not sure (x-y)(x+y) is more accurate, since it is 2 sums instead of 1. I checked with mpmath and this is what I have got:

![image](https://user-images.githubusercontent.com/54369/34096987-b3bf55d8-e3d7-11e7-8f27-bff4bebd9894.png)

Anyway we would need to change consistently everywhere...